### ISSUE TITLE
xom WARNINGS in current master-next build

### ISSUE THREAD

**Issue by moto-timo** [meta-java/meta-java#33]
Just putting these here for tracking. If you have interest in helping out, please let us know which one you are taking and send a patch to the mailing list or create a pull request. If you are aware of a patch that has been missed, please let us know.

```
WARNING: /workspace-meta-java/build/../meta-java/recipes-core/xml-commons/xom_1.2.10.bb: URL: git://github.com/codehaus/jaxen;name=jaxen uses git protocol which is no longer supported by github. Please change to ;protocol=https in the url.
```
```
WARNING: /workspace-meta-java/build/../meta-java/recipes-core/xml-commons/xom_1.2.10.bb: URL: git://github.com/codehaus/jaxen;name=jaxen does not set any branch parameter. The future default branch used by tools and repositories is uncertain and we will therefore soon require this is set in all git urls.
```

**Comment by JerryMain** [comment]
I had a patch for this that I was going to send, but it seems one is already included on master-next, or is this still an issue?

**Comment by moto-timo** [comment]
@JerryMain the patch from the mailing list which was under test on master-next is not correct. It will be dropped. Please send your patch, it will be much appreciated!

**Comment by JerryMain** [comment]
Patch is on mailing list: https://lists.openembedded.org/g/openembedded-devel/message/109531

**Comment by JerryMain** [comment]
Fixed with e13c30d7422d4f95a20d0728f5f025ae7438aa24

**Comment by JerryMain** [comment]
Fixed with e13c30d7422d4f95a20d0728f5f025ae7438aa24
